frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/HIVAoEForm.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+
+import RadioGroup from "../../../commonComponents/RadioGroup";
+import {
+  getPregnancyResponses,
+  PregnancyCode,
+} from "../../../../patientApp/timeOfTest/constants";
+import { QueriedTestOrder } from "../../QueueItem";
+import { useTranslatedConstants } from "../../../constants";
+import MultiSelect from "../../../commonComponents/MultiSelect/MultiSelect";
+import { AoeQuestionResponses } from "../TestCardFormReducer";
+
+interface HIVAoeFormProps {
+  testOrder: QueriedTestOrder;
+  responses: AoeQuestionResponses;
+  hasAttemptedSubmit: boolean;
+  onResponseChange: (responses: AoeQuestionResponses) => void;
+}
+
+const pregnancyResponses = getPregnancyResponses();
+
+export const HIVAoEForm = ({
+  testOrder,
+  responses,
+  hasAttemptedSubmit,
+  onResponseChange,
+}: HIVAoeFormProps) => {
+  const onPregnancyChange = (pregnancyCode: PregnancyCode) => {
+    onResponseChange({ ...responses, pregnancy: pregnancyCode });
+  };
+
+  const { GENDER_IDENTITY_VALUES } = useTranslatedConstants();
+
+  const onSexualPartnerGenderChange = (selectedItems: string[]) => {
+    onResponseChange({
+      ...responses,
+      genderOfSexualPartners: selectedItems,
+    });
+  };
+
+  const isPregnancyFilled = !!responses.pregnancy;
+  const isGenderOfSexualPartnersAnswered =
+    !!responses.genderOfSexualPartners &&
+    responses.genderOfSexualPartners.length > 0;
+
+  const showPregnancyError = hasAttemptedSubmit && !isPregnancyFilled;
+  const showGenderOfSexualPartnersError =
+    hasAttemptedSubmit && !isGenderOfSexualPartnersAnswered;
+
+  return (
+    <div
+      className="grid-col"
+      id={`hiv-aoe-form-${testOrder.patient.internalId}`}
+    >
+      <div className="grid-row">
+        <div className="grid-col-auto">
+          <RadioGroup
+            legend="Is the patient pregnant?"
+            name={`pregnancy-${testOrder.internalId}`}
+            onChange={onPregnancyChange}
+            buttons={pregnancyResponses}
+            selectedRadio={responses.pregnancy}
+            required={true}
+            validationStatus={showPregnancyError ? "error" : undefined}
+            errorMessage={
+              showPregnancyError && "Please answer this required question."
+            }
+          />
+        </div>
+      </div>
+      <div className="grid-row">
+        <div className="grid-col-auto">
```

Comment:
this class is causing the multiselect to change with more/less selections

https://github.com/CDCgov/prime-simplereport/assets/20211771/db04bdbe-4160-41d2-8278-c13d1a6dfe21

Reply:
@emyl3 thanks for finding this! I'm just waiting to hear back from Ryan regarding the desired sizing behavior of multiselect here

Reply:
@mehansen @jayna-SkylightDigital similar to what @emyl3 mentioned above, I noticed that those other parts of the app like Print and View details don't seem to show RSV either so we may need to prioritize a ticket for that to release RSV asap

![Screenshot 2023-11-16 095000](https://github.com/CDCgov/prime-simplereport/assets/23287037/0a9963e0-16f3-4b44-96d5-498ce0c7d98c)

![Screenshot 2023-11-16 095028](https://github.com/CDCgov/prime-simplereport/assets/23287037/d5d76a83-024c-4021-ac41-fe5bd3614f41)

![Screenshot 2023-11-16 095050](https://github.com/CDCgov/prime-simplereport/assets/23287037/1cd2b3ce-2bee-4fec-842d-83f5acc2c0dd)